cypress/integration/login.spec.js

```diff
@@ -59,7 +56,7 @@ describe('Test the authentication flow for medical adjudicators', () => {
     cy.get('[name=password]').type('secret');
     cy.get('[type="submit"]').click();
     cy.url().should('include', '/en/reports');
-    cy.get('h1').contains('Sessions');
+    cy.get('h1').contains('completed medical reports');
```

Comment:
This should use a cypress data-selector

https://docs.cypress.io/guides/references/best-practices.html

Reply:
you're right. fixed.